packages/resource-deployment/helm-charts/e2e-test-runner/templates/cron-job.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ include "e2e-test-runner.fullname" . }}
+  labels:
+    {{- include "e2e-test-runner.labels" . | nindent 4 }}
+spec:
+  schedule: {{ .Values.jobSettings.schedule | quote }}
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      activeDeadlineSeconds: {{ .Values.jobSettings.maxRuntimeSeconds }}
+      template:
+        metadata:
+          labels:
+            aadpodidbinding: {{ .Values.podPrincipalName }}
+            {{- include "e2e-test-runner.labels" . | nindent 12 }}
+        spec:
+          restartPolicy: Never
```

Comment:
do we need to restart OnFailure?

Reply:
I don't think we need to, since it runs on a schedule and we expect the job to fail if the E2E tests fail